plugin/pkg/auth/authenticator/token/anytoken/anytoken.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package anytoken
+
+import (
+	"strings"
+
+	"k8s.io/kubernetes/pkg/auth/user"
+)
+
+type AnyTokenAuthenticator struct{}
+
+func (AnyTokenAuthenticator) AuthenticateToken(value string) (user.Info, bool, error) {
+	userTokens := strings.SplitN(value, "/", 2)
```

Comment:
> from a user perspective, I'd like this to be consistent. I could see using this to "prototype" scenarios without an idp so if I could spoof an oidc handle thatd be great. Looking at the code the "split" on line 28 would probably break that

I'm open to delimiter suggestions, but `/` is a pretty good one since we know that no API resource will ever allow a slash to be in a username since that breaks the kube API server.

To my knowledge, there's no other character that guarantees that, so any other delimiter we choose would break the use-case you're describing with an identity management system.

I could also be convinced to do a "last-index-of" if we decide to take this forward.